### ISSUE TITLE
[Android]Installation error:' :react-native-maps' could not be found in project ':app'.

### ISSUE THREAD

**Issue by Yaweii** [react-community/react-native-maps#2313]
### Is this a bug report?
No

### Have you read the [Installation Instructions](https://github.com/react-community/react-native-maps/blob/master/docs/installation.md)?

Yes
### Steps to Reproduce

I am trying to install 'react-native-maps' in my react-native project for the first time.  I followed the steps for [Installation](https://github.com/react-community/react-native-maps/blob/master/docs/installation.md).
After I finished the installation process and I did a react-native run-android. I saw an error.

![image](https://user-images.githubusercontent.com/10950399/41197384-ebd7ad78-6c10-11e8-9dc7-d68f4d8d0e22.png)

I've searched on Stackoverflow and Github issues and I didn't find answers.  If I missed any open or closed tickets, I apologize first. I am getting stuck on this. Any suggestion will be appreciated!

### Expected Behavior

The app should be built successfully.

### Actual Behavior

![image](https://user-images.githubusercontent.com/10950399/41197392-38687618-6c11-11e8-9dd5-d2327d6f8a01.png)

### Reproducible Demo

(Paste the link to an example project and exact instructions to reproduce the issue.)

**Comment by Yaweii** [comment]
So I did some research and fixed this issue. Let me explain a little bit and it might be helpful for people who face the same issue.

1. 

Even if you followed the installation docs step by step, there is a chance you will still face some issues. that's because the latest react-native-maps is using syntax for **Gradle 3** and your Gradle is probaby 2. Here is a related issue [2188](https://github.com/react-community/react-native-maps/issues/2188) 

- How to check you gradle? Go to android/build.gradle, under dependencies you will find your gradle. Mine for example is 2.2.3.

2. 
After you decide either upgrade your **Gradle** to 3, or hack the react-native-maps pacakge.  You will be fine.
3. 
If you still facing some issues. Here are some issues I found useful and helped me with my situation.
[1678](https://github.com/react-community/react-native-maps/issues/1678)
[1814](https://github.com/react-community/react-native-maps/issues/1814)
